DigitalLearningSolutions.Data.Tests/DataServices/CentresDataServiceTests.cs

```diff
@@ -301,5 +301,24 @@ public void SetCentreAutoRegistered_should_set_AutoRegistered_true()
                 transaction.Dispose();
             }
         }
+
+        [Test]
+        public void GetAllCentreSummaries_returns_all_summary_details_and_reference_data()
+        {
+            using var transaction = new TransactionScope();

```

Comment:
We are using a transaction here, but we are not modifying any records, so it can probably get gotten rid of.

Reply:
Indeed, good spot.